rustler_mix/lib/rustler/compiler.ex

```diff
@@ -43,8 +43,8 @@ defmodule Rustler.Compiler do
   end
 
   defp make_base_command(:system), do: ["cargo", "rustc"]
+  defp make_base_command({:system, channel}), do: ["cargo", channel, "rustc"]
```

Comment:
This should be documented in [rustler.ex](https://github.com/rusterlium/rustler/blob/master/rustler_mix/lib/rustler.ex#L22)

Reply:
Added